app/src/bcsc-theme/features/modal/DeviceInvalidated.tsx

```diff
@@ -12,6 +14,8 @@ import { SystemModal } from './components/SystemModal'
  */
 export const DeviceInvalidated = (): JSX.Element => {
   const { t } = useTranslation()
+  const route = useRoute()
+  const caseType = (route.params as { caseType?: BCSCReason })?.caseType
```

Comment:
suggestion: you can type the props for `params` to prevent having to cast.

see EmailConfirmationScreen.tsx and VerifyWebViewScreen.tsx